WordPress/src/main/res/layout/login_magic_link_request_screen.xml

```diff
@@ -6,21 +6,37 @@
         android:id="@+id/toolbar"
         layout="@layout/toolbar_login" />
 
-    <org.wordpress.android.widgets.WPNetworkImageView
-        android:id="@+id/gravatar"
+    <FrameLayout
+        android:id="@+id/avatar_container"
         android:layout_width="@dimen/avatar_sz_login"
         android:layout_height="@dimen/avatar_sz_login"
-        android:layout_centerHorizontal="true"
         android:layout_below="@+id/toolbar"
-        android:layout_marginTop="@dimen/margin_extra_extra_large"/>
+        android:layout_marginTop="@dimen/margin_extra_extra_large"
+        android:layout_marginBottom="@dimen/margin_extra_large"
+        android:layout_centerHorizontal="true">
+
+        <org.wordpress.android.widgets.WPNetworkImageView
+            android:id="@+id/gravatar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
+
+        <ProgressBar
+            android:id="@+id/avatar_progress"
+            style="@android:style/Widget.Holo.ProgressBar"
```

Comment:
Oh, no particular reason.

I changed it to `@style/Widget.AppCompat.ProgressBar` in 4de3171. `Widget.Material.ProgressBar` is not available in the compat lib yet AFAICT.